src/main/java/net/tiagofar78/prisonescape/game/phases/Finished.java

```diff
@@ -1,5 +1,20 @@
 package net.tiagofar78.prisonescape.game.phases;
 
-public class Finished {
+public class Finished extends Phase {
 
+    private boolean _clockStopped;
+    
+    public Finished (boolean clockStopped) {
+        _clockStopped = clockStopped;
+    }
+
+    @Override
+    public Phase next() {
+        return new Waiting(_clockStopped);
+    }
+
+    @Override
+    public boolean isClockStopped() {
+        return _clockStopped;
```

Comment:
Usually when using State desing pattern functions are expected to have already defined values instead of a variable. In this case it shoud return false because in the finished phase there is no need to keep the game clock running.